yt/visualization/plot_modifications.py

```diff
@@ -3238,10 +3238,10 @@ def __init__(
         field_x,
         field_y,
         texture=None,
-        kernellen=50.0,
+        kernellen=50,
         lim=(0.5, 0.6),
         cmap="binary",
-        alpha=0.8,
+        alpha=0.5,
```

Comment:
I found that a lower default alpha (opacity) made more sense with the correct implementation where "ink" is more uniformly distributed across the image so it's easier to hide the background completely if you're not careful.